http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -83,11 +80,18 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       )
       _ <- F.raiseWhen(!hasPermission)(ForbiddenError(userInfo.userEmail))
 
-      _ <- req.appType match {
+      enableIntraNodeVisibility <- req.appType match {
+        case AppType.Galaxy | AppType.HailBatch | AppType.Wds => F.pure(false)
```

Comment:
Up to you, but I would not necessarily worry about WDS or HailBatch since they are only implemented for Azure